keyboards/horrortroll/nyx/info.json

```
@@ -0,0 +1,229 @@
+{
+    "keyboard_name": "Nyx",
+    "manufacturer": "HorrorTroll",
+    "maintainer": "HorrorTroll",
+    "usb": {
+        "vid": "0x7516",
+        "pid": "0x5005",
+        "device_version": "0.0.1"
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["GP3", "GP23", "GP2", "GP1", "GP0", "GP4", "GP17", "GP11", "GP10", "GP19", "GP9", "GP8", "GP13", "GP15", "GP14"],
+        "rows": ["GP24", "GP25", "GP18", "GP12", "GP16"]
+    },
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true,
+        "rgb_matrix": true
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP22"
+    },
+    "rgb_matrix": {
+        "driver": "WS2812",
```

migrated from `config.h`
```suggestion
        "driver": "WS2812",
        "max_brightness": 200,
```

---

Thanks. I've just pushing new commit.